src/components/Template/templateChangelogData.ts

```diff
@@ -9,6 +9,15 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Functionality", "Styles"],
+    notes: [
+      "Major update to component and styles based on the [Template TAD](https://docs.google.com/document/d/1vZJX7Y-DnEM44-iWw5qoXGdxqavHEN6prhr-YDTNr0o/edit?pli=1&tab=t.0).",
```

Comment:
This doesn't render as a link in storybook since it can't parse mdx strings. You can leave out the link.

Reply:
<img width="498" alt="Screenshot 2024-12-06 at 11 08 34 AM" src="https://github.com/user-attachments/assets/bd3d637e-a99d-4e18-9277-2b4bd7fbbe7c">